src/Frontend/Components/AggregatedAttributionsPanel/WorkerAccordionPanel.tsx

```
@@ -41,6 +41,14 @@ export function WorkerAccordionPanel(
   ] = useState<AttributionIdsWithCountAndResourceId>(
     emptyAttributionIdsWithCountAndResourceId
   );
+  const resourceDetailsTabsWorkers = useContext(WorkersContext);
+
+  const worker =
+    props.title === PackagePanelTitle.ContainedExternalPackages
+      ? resourceDetailsTabsWorkers.containedExternalAttributionsAccordionWorker
+      : props.title === PackagePanelTitle.ContainedManualPackages
+      ? resourceDetailsTabsWorkers.containedManualAttributionsAccordionWorker
+      : null;
```

if null we could throw an error